README.md

```diff
@@ -419,6 +419,60 @@ RESULTS
 |---------|------------------------|--------------------------------|--------------------------------|---------------------------------------------------------------------------|
 ```
 
+##### Showing Maintainers Annotation
+
+To see maintainers annotations for each check, use the `--show-ma` option. The annotations are configured as labels but parsed to a human-readable format in the output.
+
+To add annotations to your repository, create a `scorecard.yml` file in the root of your repository.
+
```

Comment:
in an effort to keep the README shorter/higher-level, we may want to move everything here and below to a separate README for detailed maintainer annotation instructions.

Reply:
Done.